lib/src/itunes_search_api.dart

```diff
@@ -32,14 +32,16 @@ class ITunesSearchAPI {
   /// ```lookupURLByBundleId('com.google.Maps');```
   /// ```lookupURLByBundleId('com.google.Maps', country: 'FR');```
   Future<Map?> lookupByBundleId(String bundleId,
-      {String? country = 'US', bool useCacheBuster = true}) async {
+      {String? country = 'US',String? language = 'en', bool useCacheBuster =
+      true}) async {
     assert(bundleId.isNotEmpty);
     if (bundleId.isEmpty) {
       return null;
     }
 
     final url = lookupURLByBundleId(bundleId,
-        country: country ?? '', useCacheBuster: useCacheBuster)!;
+        country: country ??= '', language: language ?? '', useCacheBuster:
```

Comment:
@ibraheemalayan There seems to be an extra character added to this existing code `country: country ??= ''`. The `=` was added. Was there a reason for thiat?

Reply:
Looks like it is a mistake but it doesn't affect the code.

Reply:
I can merge this after that extra character is removed.

Reply:
I did the change